packages/web/docs/src/components/ecosystem-page/components.tsx

```diff
@@ -0,0 +1,93 @@
+import { DecorationIsolation, Heading, ProductCard, PRODUCTS } from '@theguild/components';
+import EcosystemPageContent from './content.mdx';
+import { EcosystemPageNavH2 } from './ecosystem-page-nav-h2';
+
+export const components = {
+  EcosystemHeader: (props: React.HTMLAttributes<HTMLDivElement>) => (
+    <header
+      className="relative isolate flex max-w-[90rem] flex-col items-center gap-6 overflow-visible rounded-3xl bg-blue-400 px-4 py-6 text-center sm:py-12 md:gap-8 lg:py-24 [&>h1]:max-w-[800px] [&>p]:max-w-[520px]"
+      {...props}
+    >
+      <span className="font-medium">The Ecosystem</span>
+      {props.children}
+      <CrossDecoration />
+      <nav className="absolute top-full grid -translate-y-1/2 grid-flow-col rounded-2xl bg-blue-400 [grid-auto-columns:1fr]">
+        <EcosystemPageContent components={ecosystemPageNav} />
+      </nav>
+    </header>
+  ),
+  h1: (props: React.HTMLAttributes<HTMLHeadingElement>) => <Heading as="h1" size="xl" {...props} />,
+  h2: (props: React.HTMLAttributes<HTMLHeadingElement>) => (
+    <Heading as="h2" size="xs" className="mb-4 mt-24" {...props} />
+  ),
+  p: (props: React.HTMLAttributes<HTMLParagraphElement>) => (
+    <p className="text-green-800" {...props} />
+  ),
+  ul: (props: React.HTMLAttributes<HTMLUListElement>) => {
+    return (
+      <ul
+        className="mt-5 grid grid-cols-4 gap-5 overflow-x-auto p-4 last-of-type:mb-24"
+        {...props}
+      />
+    );
+  },
+  li: (props: React.LiHTMLAttributes<HTMLLIElement>) => {
+    const productName = String(props.children)
+      .toUpperCase()
+      .replace(' ', '_') as keyof typeof PRODUCTS;
+
+    const product = PRODUCTS[productName];
+    if (!product) {
+      throw new Error(`Product ${productName} is missing`);
+    }
+
+    return <ProductCard as="li" product={product} className="h-[222px]" {...props} />;
+  },
+};
+
+const ecosystemPageNav = {
+  ...Object.fromEntries(Object.keys(components).map(key => [key, () => null])),
```

Comment:
```suggestion
  // ...
  ...Object.fromEntries(Object.keys(components).map(key => [key, () => null])),
```
just add comment why you hide `components`